docs/getting-started-monitoring.md

```diff
@@ -278,6 +293,15 @@ First, you need to install the following tools used in this guide.
                  kubectl get secrets scalar-monitoring-grafana -n monitoring -o jsonpath='{.data.admin-password}' | base64 -d
                  ```
 
+1. Access log browser
+   - Go to Grafana http://localhost:3000
+   - Move to `Configuration` and choose `Data Sources`
+   - Click `Add data source`
+   - Select `Loki`
+   - Input `http://loki-stack:3100` to URL
```

Comment:
```suggestion
   - Input `http://scalar-logging-loki:3100` to URL
```
As I described another comment, I think it is better to use more understandable `release name` that users can know the purpose of deployment is Logging for Scalar products.

Reply:
Thanks for the advice.
I revised it in https://github.com/scalar-labs/helm-charts/pull/87/commits/eb349d4503420bc0cdcf55ac9bef9de68a97ea80